warehouse/templates/api/simple/detail.html

```diff
@@ -20,7 +20,7 @@
   <body>
     <h1>Links for {{ name }}</h1>
     {% for file in files -%}
-    <a href="{{ file.url }}#sha256={{ file.hashes.sha256 }}" {% if file.get('requires-python') %}data-requires-python="{{ file['requires-python'] }}" {% endif %}{% if file.yanked %}data-yanked="{% if file.yanked is string %}{{ file.yanked }}{% endif %}" {% endif %}{% if file['data-dist-info-metadata'] %}data-dist-info-metadata="sha256={{ file['data-dist-info-metadata']['sha256'] }}"{% endif %}>{{ file.filename }}</a><br />
+    <a href="{{ file.url }}#sha256={{ file.hashes.sha256 }}" {% if file.get('requires-python') %}data-requires-python="{{ file['requires-python'] }}" {% endif %}{% if file.yanked %}data-yanked="{% if file.yanked is string %}{{ file.yanked }}{% endif %}" {% endif %}{% if file['data-dist-info-metadata'] %}data-dist-info-metadata="sha256={{ file['data-dist-info-metadata']['sha256'] }}" data-core-metadata="sha256={{ file['data-dist-info-metadata']['sha256'] }}"{% endif %}>{{ file.filename }}</a><br />
```

Comment:
These keys probably need updated to say ``file["core-metadata"]`` if I remember the way this works correctly.

Reply:
the object part comes back key'd as 'data-dist-info-metadata' which is internal to the implementation.

we can remap them internally as part of this PR!

Reply:
ope, we already did! we just send both from `warehouse.packaging.utils._simple_detail` we can reference either here. I'll update to the new canonical source :)

Reply:
Ah missed the previous key in `_simple_detail` was still there, probably worth to use the new name and just remove the old name completely to keep it from getting out of date or something?

Reply:
9de3b7b2b76b4d480144de0b0604781ab025655d